### PR TITLE
Update NamedCSVImporter.java

### PR DESCRIPTION
**Pull request by cri-s**
Removed trace logging. 
This makes initially sense, but after time are hard to follow in case of debug.

### CODE REVIEW

**Comment by vonnieda** [comment]
Please use the template provided when submitting a pull request.

**Comment by cri-s** [comment]
It is not a feature request and don't resolve any issue, just code cleanup.
Because that i have found inappropiate using the template that is for
the two cases,
or i have interpretate the template wrong ?

2017-04-14 18:22 GMT+02:00, Jason von Nieda <notifications@github.com>:

**Comment by vonnieda** [comment]
The pull request template is different from the issue template. It is shown automatically when you start a pull request on Github: https://github.com/openpnp/openpnp/blob/develop/PULL_REQUEST_TEMPLATE.md
